precommit

```diff
@@ -2,6 +2,6 @@
 docker exec tcf_django autopep8 --in-place --aggressive --aggressive -r tcf_website tcf_core
 # Lint.
 docker exec tcf_django pylint --load-plugins pylint_django --django-settings-module=tcf_core.settings tcf_website tcf_core
-docker exec tcf_django node_modules/.bin/eslint --fix tcf_website/static/**/*.js -c .config/.eslintrc.yml
+docker exec tcf_django node_modules/.bin/eslint --fix 'tcf_website/static/**/*.js' -c .config/.eslintrc.yml
```

Comment:
I realized that the default value of `--ext` is `.js`, so I thought it would be shorter and more intuitive to use the directory name instead: `docker exec tcf_django node_modules/.bin/eslint --fix -c .config/.eslintrc.yml tcf_website/static`.
(changed the order since it's written in this order in the documentation: `eslint [options] file.js [file.js] [dir]`)